style.css

```diff
@@ -67,7 +63,6 @@ header p {
 }
 
 .container {
-  max-width: 80%;
   margin: 0 auto;
 }
 
```

Comment:
- [ ] Kindly fix your margin on different elements
- [ ] Hamburger button should not be visible on desktop version since this is a desktop version
- [ ] Your background image looks very stretch, maybe you can try another property to make it looks better

![image](https://user-images.githubusercontent.com/93630700/214734537-48b04e25-1b99-4ea6-8725-340aafcb9391.png)